pipeline/extract.py

```diff
@@ -30,11 +30,13 @@ def get_sales_information() -> dict:
     return sales_data
 
 
-def main_extract():
+def main_extract() -> dict:
     """Runs all the functions necessary for
     extraction in one go."""
```

Comment:
Maybe a better docstring would be "Main function to extract data from the Bandcamp API and returns a dictionary containing raw data from the API."